workflow/metrics/metrics.go

```diff
@@ -205,6 +213,31 @@ func (m *Metrics) CronWorkflowSubmissionError() {
 	m.errors[ErrorCauseCronWorkflowSubmissionError].Inc()
 }
 
+func (m *Metrics) WorkerBusy(workerType string) {
+	m.mutex.Lock()
```

Comment:
I think these mutex may create contention in the VM - doesn't the gauge already have these?

Reply:
Sorry, I'm not too sure what you mean. These mutexes are needed to keep concurrency issues between writing/updating metrics and metrics being scraped. While individual metrics may be concurrent safe, the collection of metrics and the metrics scraper are not without these locks. They are present in all other functions in this file.